src/WebJobs.Extensions.DurableTask/ContextImplementations/DurableCommonContext.cs

```diff
@@ -372,32 +368,64 @@ internal virtual void ThrowIfInvalidAccess()
                 {
                     // If this were not a replay, then the orchestrator/activity/entity function trigger would have already
                     // emitted a FunctionFailed trace with the full exception details.
-                    this.Config.TraceHelper.FunctionFailed(
-                        this.Config.Options.HubName,
-                        functionName,
-                        this.InstanceId,
-                        operationId,
-                        operationName,
-                        reason: $"(replayed {exception.GetType().Name})",
-                        functionType: functionType,
-                        isReplay: true);
+
+                    if (functionType == FunctionType.Entity)
+                    {
+                        this.Config.TraceHelper.OperationCompleted(
```

Comment:
Should this be `OperationFailed`?

Reply:
It's `OperationCompleted` with `failed:true`. I used the same call since all the parameters align.

Reply:
I think I will revert that to two separate calls so it is more in line with the rest of the tracing code.